### PR TITLE
Gis locations pcode remapping

### PR DESCRIPTION
**Pull request by csabadenes**

### CODE REVIEW

File: src/etools/applications/locations/tasks.py [hunk 1]
```diff
@@ -56,34 +70,56 @@ def create_location(pcode, carto_table, parent, parent_instance,
...
         try:
             location.save()
         except IntegrityError:
             logger.exception('Error while saving location: %s', site_name)
-            return False, sites_not_added, sites_created, sites_updated
-
-        sites_updated += 1
+            return False, sites_not_added, sites_created, sites_updated, sites_remapped
+
+        if should_remap is True:
```
Comment (ntrncic) [inline src/etools/applications/locations/tasks.py]: don't need to an else statement just if: then after if block the rest of code

File: src/etools/applications/locations/models.py [hunk 2]
```diff
@@ -94,6 +94,13 @@ class Location(MPTTModel):
         blank=True,
     )
     point = models.PointField(verbose_name=_("Point"), null=True, blank=True)
+
+    # fields where previous data is backed up on pcode remap
+    date_remapped = models.DateTimeField(verbose_name=_('Remap date and time'), null=True)
```
Comment (ntrncic) [inline src/etools/applications/locations/models.py]: move to a seperate model

**Comment by csabadenes** [comment]
@ntrncic I think this one can be closed

**Comment by csabadenes** [comment]
@domdinicola , i am closing this request, the PR was redone in another branch, it's already merged and closed.

**Pull request closed without merging**